### PR TITLE
Demo first commit

### PR DESCRIPTION
**Pull request by yogevbar**

### CODE REVIEW

File: IBandPlayerDemo/Assets.xcassets/iconHDOn.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "iconHDOn.png",
```
Comment (moshest) [inline IBandPlayerDemo/Assets.xcassets/iconHDOn.imageset/Contents.json]: Do not support plain videos on the demo, remove all HD and 360 icons

File: IBandPlayerDemo/Entity.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
+//
+//  Entitie.h
+//  IBand.live
+//
+//  Created by Yogev Barber on 16/08/2017.
+//  Copyright © 2017 Yogev Barber. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```
Comment (moshest) [inline IBandPlayerDemo/Entity.h]: `Entity` should not be on the demo.

File: IBandPlayerDemo/Entity.m [hunk 3]
```diff
@@ -0,0 +1,31 @@
+//
+//  Entitie.m
+//  IBand.live
+//
+//  Created by Yogev Barber on 16/08/2017.
+//  Copyright © 2017 Yogev Barber. All rights reserved.
+//
+
+#import "Entity.h"
```
Comment (moshest) [inline IBandPlayerDemo/Entity.m]: `Entity` should not be on the demo.

**Review by moshest — CHANGES_REQUESTED** [review]

File: IBandPlayerDemo/Assets.xcassets/iconHDOn.imageset/Contents.json (hunk 1 above)
Comment (yogevbar) [reply IBandPlayerDemo/Assets.xcassets/iconHDOn.imageset/Contents.json]: done

File: IBandPlayerDemo/Entity.h (hunk 2 above)
Comment (yogevbar) [reply IBandPlayerDemo/Entity.h]: done.

File: IBandPlayerDemo/Entity.m (hunk 3 above)
Comment (yogevbar) [reply IBandPlayerDemo/Entity.m]: done.

**Review by moshest — APPROVED** [review]

**Pull request merged**